openedx/features/course_duration_limits/access.py

```diff
@@ -69,10 +68,6 @@ def get_user_course_expiration_date(user, course):
     if enrollment is None or enrollment.mode != 'audit':
         return None
 
-    # if the user is a beta tester their access should not expire
-    if CourseBetaTesterRole(course.id).has_user(user):
```

Comment:
i moved this to enabled_for_enrollment.  
@cpennington suggested on the holdback PR consolidating various checks in that function